email-auditing-api.yaml

```diff
@@ -0,0 +1,253 @@
+    get:
+      description: |
+        Lists all the Experience programs or Ad Hoc surveys to audit emails from.
+      operationId: listPrograms
+      parameters:
+        - description: Position of the Experience program or Ad Hoc survey in the list.
+          in: query
+          name: offset
+          schema:
+            type: integer
+        - description: Maximum number of Experience programs or Ad Hoc surveys to retrieve in the list.
+          in: query
+          name: limit
+          schema:
+            type: integer
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/programs"
+          description: Request processed as expected.
+        "401":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+          description: Unauthorized.
+        "403":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+          description: Forbidden.
+        "500":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+          description: Internal server error.
+components:
+  responses:
+    "202":
+      content: {}
+      description: Accepted
+    "304":
+      content: {}
+      description: Not modified
+    "400":
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/input-validation-error"
+      description: Invalid request
+    "401":
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/error"
+      description: Unauthorized.
+    "403":
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/error"
+      description: Forbidden.
+    "404":
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/error"
+      description: Object not found.
+    "406":
+      content: {}
+      description: Not acceptable.
+    "409":
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/error"
+      description: Request conflict with current state of the server.
+    "415":
+      content: {}
+      description: Not supported.
+    "500":
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/error"
+      description: Internal server error.
+  schemas:
+    programs:
+      example:
+        _total: 10
+        items:
+          - name: Experience Program #1
+            id: 93d3f221-67f6-4f51-b417-4f5ab194728b
+          - name: Ad Hoc Survey #1
+            id: f3c689fb-7e01-4e88-9642-79079fb5d4de
+      properties:
+        items:
+          items:
+            $ref: "#/components/schemas/program"
+          type: array
```

Comment:
`type` needs to come first, before `items` or `format`.  The `type` defines everything else.